### DISCUSSION TITLE
Welcome to DSC-TLink Discussions!

### DISCUSSION THREAD

**Discussion by BrianHumlicek** [BrianHumlicek/DSC-TLink#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kamitron** [comment]
Hello!

I am very happy with this project. I would like to try it. The hardware conditions are given. I am a beginner in the world of Home assistant and Github. Could you please help me step by step? How and where should the files be copied and what should be set in the home assistant?

Thank you for your help!

**Reply by BrianHumlicek** [reply]
Currently, there are no home assistant components in this project.  It is intended to be more of a proof of concept at this point.

Regarding getting involved, this is not a beginner project at this point.  Perhaps in the future it will require less domain knowledge, but for now it’s pretty hardcore.
